doc/cephfs/administration.rst

```diff
@@ -82,6 +82,16 @@ file system.  If any files have layouts for the removed data pool, the file
 data will become unavailable. The default data pool (when creating the file
 system) cannot be removed.
 
+::
+
+    fs rename <file system name> <new file system name> [--yes-i-really-mean-it]
+
+Rename a Ceph file system. This also changes the application tags on the data
+pools and metadata pool of the file system to the new file system name.
+The cephx IDs authorized to the old file system name need to be re-authorized
```

Comment:
nit:
```suggestion
The CephX IDs authorized to the old file system name needs to be re-authorized
```

Reply:
s/need/needs/  is incorrect.  The subject of the sentence is IDs. Using "need" adheres to subject-verb agreement rule.

I did make s/cephx/CephX/  But I'm unsure. On 'git blame' almost all of this change is in CephFS docs, and attributed to you. What prompted you to start making this change? In the glossary and Red Hat docs I still see cephx being used. https://docs.ceph.com/en/latest/glossary/#ceph-glossary

Reply:
>  did make s/cephx/CephX/ But I'm unsure. On 'git blame' almost all of this change is in CephFS docs, and attributed to you. What prompted you to start making this change? In the glossary and Red Hat docs I still see cephx being used. https://docs.ceph.com/en/latest/glossary/#ceph-glossary

I picked it from docs but can't recall the specific page. Right now, I agree that docs are a mix of "Cephx" and "cephx". "CephX" is almost only used in code.